packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -40,6 +39,12 @@ import {RESOLVED_PROMISE} from './express'
 import http from 'http'
 import https from 'https'
 import {proxyConfigs, updatePackageMobify} from '../../utils/ssr-shared'
+import {
+    startsWithMobify,
+    getProxyPathBase,
+    getHealthCheckPath,
+    getSLASPrivateProxyPath
+} from '../../utils/ssr-namespace-paths'
```

Comment:
The file name is called "blah-blah-paths" which would imply that it has *drum roll* a bunch of paths in it. But we actually have a util and other paths, which means we had to postfix all the paths with "Path". I wonder if it would make sense to move that util so that the file name is true. Otherwise we might want to think about renaming this file, but I think that might be a lesser of the 2 solutions.

Reply:
Mentioned in another thread above, but I have moved the util function into `build-remote-server` so all this file exports now are paths.